qiskit/providers/ibmq/experiment/experimentservice.py

```diff
@@ -478,6 +479,9 @@ def upload_plot(
             plot: Name of the plot file or plot data to upload.
             plot_name: Name of the plot. If ``None``, the plot file name, if
                 given, or a generated name is used.
+            sync_upload: By default the server will upload the plot file
+                to backend storage asynchronously. Set this to False to use
+                that behavior and not block the upload.
```

Comment:
```suggestion
            sync_upload: If ``True``, the plot will be uploaded synchronously. 
                Otherwise the upload will be asynchronous. 
```